include/exadg/solvers_and_preconditioners/multigrid/multigrid_preconditioner_base.cpp

```diff
@@ -861,7 +891,8 @@ MultigridPreconditionerBase<dim, Number>::do_initialize_transfer_operators(
   {
     auto tmp = std::make_shared<MGTransferGlobalCoarsening<dim, MultigridNumber, VectorTypeMG>>();
 
-    tmp->reinit(*mapping, matrix_free_objects, constraints, constrained_dofs, dof_index);
+    MappingQGeneric<dim> mapping_dummy(1);
+    tmp->reinit(mapping_dummy, matrix_free_objects, constraints, constrained_dofs, dof_index);
```

Comment:
@peterrum can I remove Mapping from this interface, or will it be needed in the future?

Reply:
the `reinit()` function is not part of the interface of the base class `MGTransfer`

Reply:
It is not needed. I think we wrote this like this to be consistent with the reinit function of the other transfer operator. I don't mind if you'd remove it.

Reply:
removed it, and also another parameter